### ISSUE TITLE
Projection of 3D bounding box to individual frame

### ISSUE THREAD

**Issue by chanhee-luke** [smartscenes/multiscan#17]
Hi, I'm trying to project the 3D bounding box to the individual frame.

I'm using the transform matrix (following the doc https://3dlg-hcvc.github.io/multiscan/read-the-docs/dataset/files/acquired.html#camera-parameters) and the intrinsic matrix to do the calculation but the results doesn't seem quite right. Could you help me on this? Thanks!

Best,
Luke

**Comment by xjj1999** [comment]
Hi, How can I get the download script? I note that the application form is no longer valid.

https://docs.google.com/forms/d/e/1FAIpQLSfksFtks9YHMeQQWjZjfNbNU4bhRx0knyJ_S-OdJ-vdi2pjBw/closedform

**Comment by Qi-Zhangyang** [comment]
Have you finished the projection?

I am also doing the same thing but it does not align.

**Comment by chanhee-luke** [comment]
@Qi-Zhangyang  No, it did not align so I just gave up. Idk what's going on with the annotations.
